tests/e2e/go.mod

```diff
@@ -10,7 +10,7 @@ require (
 	k8s.io/api v0.28.9
 	k8s.io/apimachinery v0.28.9
 	k8s.io/client-go v0.28.9
-	k8s.io/kubernetes v1.29.7
```

Comment:
Is this version downgrade required? It's also affecting other dependency versions.

Reply:
There was a recent PR where the version was upgraded casing compilation failure in the e2e test files and hence we need to downgrade it to a version where the tests compile

Reply:
@tdawe Yes the the version 1.29.7 was failing compiling e2e because K8s updated few other packages where our code wasnt complaint. We would need to change our code accordingly to match 1.29 k8s soon